pydantic_ai_slim/pydantic_ai/models/groq.py

```diff
@@ -79,6 +79,7 @@
     'llama-3.2-3b-preview',
     'llama-3.2-11b-vision-preview',
     'llama-3.2-90b-vision-preview',
+    'moonshotai/kimi-k2-instruct',
```

Comment:
We should update `GroqProvider.model_profile` to handle the `moonshotai/` prefix and call `moonshotai_model_profile` (that was introduced in the other PR but you can copy it in here as well, they shouldn't conflict)

Reply:
done!